open_mastr/utils/helpers.py

```diff
@@ -282,3 +287,73 @@ def session_scope(engine):
         raise
     finally:
         session.close()
+
+
+def technology_input_harmonisation(technology, api_data_types):
```

Comment:
Maybe we could put that function to helpers.py ?

Reply:
It is in helpers.py, isn't it?

Reply:
Ah sorry, I got confused.